Remotion/Web/Core/IResourceUrlFactory.cs

```diff
@@ -37,7 +37,7 @@ public interface IResourceUrlFactory
     ///     &lt;resource root&gt;/&lt;definingType.Assembly&gt;/&lt;ResourceType&gt;/relativeUrl.
     ///   </para><para>
     ///     The <b>resource root</b> is loaded from the application configuration,
-    ///     <see cref="Remotion.Web.Configuration.WebConfiguration.Resources">WebConfiguration.Resources</see>, and 
+    ///     <see cref="ResourceRoot"/>, and
```

Comment:
since we're touching it, merge those two lines because the new code is short enough?